### PR TITLE
RCM-225: & 223 committing python tutorial code to correct repositoryiy

### PR DESCRIPTION
**Pull request by jgrabenstein**

### CODE REVIEW

File: Intro01/Python/src/main/rapture/dayOneDemo.csv [hunk 1]
```diff
@@ -0,0 +1,141 @@
+series_type,provider,index_id,frequency,price_type,date,index_price
```
Comment (scarabus) [inline Intro01/Python/src/main/rapture/dayOneDemo.csv]: File name has changed. It's now introDataInbound.csv

File: Intro01/Python/src/main/rapture/App.py [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+		#Add price_types to the master json/dict
+		for x in indxID:
+			indx_idUpdate.update({x:priceTypeDict})
+		Order['index_id'] = indx_idUpdate
+		
+		#CREATE DICTIONARYS OF DATES/PRICES BY INDEX_ID/PRICE_TYPE
+		USG = {}
+		EMN = {}
+		LAK = {}
+		AUD = {}
```
Comment (scarabus) [inline Intro01/Python/src/main/rapture/App.py]: Hmm. Can't we do a generic version that doesn't make assumptions about the data values? 
It's one thing to hard code column names, but I'm uncomfortable about making it so specific.

Comment (jgrabenstein) [inline Intro01/Python/src/main/rapture/App.py]: Agreed i'm not a fan of it either. I'll re-engineer this bit

**Comment by scarabus** [comment]
For some reason I don't have write access to this repo, but I approve.

**Pull request merged**